components/tyk-dashboard/values.yaml

```diff
@@ -109,7 +109,40 @@ global:
 
     # Enables SSL for Redis connection. Redis instance will have to support that.
     # Default value: false
-    # useSSL: true
+    useSSL: false
+
+    # Set this to `true` to tell Pump to ignore database's cert validation.
```

Comment:
Shouldn't be `dashboard`?

Reply:
this is not clear

Reply:
I think this file is related to Dashboard, not Pump.
```
# Set this to `true` to tell Dashboard to ignore database's cert validation.
```